strabo/static/js/popup.js

```diff
@@ -88,7 +88,7 @@ while making sure that photoswipe is not opened twice.
 The delay is the second argument to window.setTimeout in units of milliseconds. This can be played with,
 but very small values like 1 do not work very well.
 */
-function set_flickety_click(){
+function set_flickity_click(){
```

Comment:
I read your comment above re: why you did this....but why did adding this delay work?! I don't have an android so I can't test it but I'm baffled by the problem/solution here.